src/tuf.rs

```diff
@@ -28,7 +28,7 @@ pub struct Tuf<D: DataInterchange> {
 impl<D: DataInterchange> Tuf<D> {
     /// Create a new `TUF` struct from a known set of pinned root key ids that are used to verify
     /// the signed metadata.
-    pub fn from_root_keyids_pinned<'a, I>(
+    pub fn with_root_with_pinned_keyids<'a, I>(
```

Comment:
You can have this delegate to `from_root_with_trusted_root_keys` and then inline this into client.

Reply:
I lifted this logic up into `Client`.